src/coreclr/jit/compiler.h

```diff
@@ -16,6 +16,8 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 */
 
+// dummy change to trigger formatting job
+
 /*****************************************************************************/
```

Comment:
Formatting jobs are green, undoing the dummy change.
```suggestion
/*****************************************************************************/
```